packages/packages.csv

```diff
@@ -156,7 +156,7 @@ Microsoft.Extensions.Configuration.Abstractions,ship,include
 Microsoft.Extensions.Configuration.AzureKeyVault,ship,include
 Microsoft.Extensions.Configuration.Binder,ship,include
 Microsoft.Extensions.Configuration.CommandLine,ship,include
-Microsoft.Extensions.Configuration.DockerSecrets,ship,include
+Microsoft.Extensions.Configuration.DockerSecrets,noship,include
```

Comment:
noship,include is a strange combination. We don't want to ship this package but want to include it in the runtime store?